.env.examples

```
@@ -13,3 +13,5 @@ GEMINI_API_KEY=
 NODEMAILER_EMAIL=
 
 NODEMAILER_PASSWORD=
+
+NEXT_FINNHUB_PUBLIC_API_KEY=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)